modules/objdetect/include/opencv2/objdetect/aruco_detector.hpp

```diff
@@ -108,9 +109,23 @@ struct CV_EXPORTS_W_SIMPLE DetectorParameters {
     /** @brief default value CORNER_REFINE_NONE */
     CV_PROP_RW int cornerRefinementMethod;
 
-    /// window size for the corner refinement process (in pixels) (default 5).
+    /** @brief maximum window size for the corner refinement process (in pixels) (default 5).
+     *
+     * The window size may decrease if the ArUco marker is too small, check minCornerRefinementWinSizeInPin.
+     * cornerRefinementWinSize is recalculated for each marker as:
+     * cornerRefinementWinSize = min(cornerRefinementWinSize, averageArucoPinSize*minCornerRefinementWinSizeInPin)
+     */
     CV_PROP_RW int cornerRefinementWinSize;
 
+    /** @brief minimum allowed window size for the corner refinement process (in pins of ArUco marker) (default 0.3).
+     *
+     * cornerRefinementWinSize is recalculated for each marker as:
+     * cornerRefinementWinSize = min(cornerRefinementWinSize, averageArucoPinSize*minCornerRefinementWinSizeInPin)
+     * In the case of markers located far from each other, it may be useful to increase the value of the parameter to 0.4-0.5.
+     * In the case of markers located close to each other, it may be useful to decrease the parameter value to 0.1-0.2.
+     */
+    CV_PROP_RW float minCornerRefinementWinSizeInPin;
```

Comment:
I propose to name the field as `relativeCornerRefinmentWinSize`. And add information about avarage Aruco pin size and logic to the description.

Reply:
renamed to `relativeCornerRefinmentWinSize`:

```
    /** @brief minimum allowed window size for the corner refinement process (in pins of ArUco marker) (default 0.3).
     *
     * cornerRefinementWinSize is recalculated for each marker as:
     * cornerRefinementWinSize = min(cornerRefinementWinSize, averageArucoModuleSize*relativeCornerRefinmentWinSize),
     * where averageArucoModuleSize is average module size of ArUco marker in pixels.
     * (ArUco marker is composed of black and white modules)
     * In the case of markers located far from each other, it may be useful to increase the value of the parameter to 0.4-0.5.
     * In the case of markers located close to each other, it may be useful to decrease the parameter value to 0.1-0.2.
     */
    CV_PROP_RW float relativeCornerRefinmentWinSize;
```